studio-docs/source/federated-graphs.mdx

```diff
@@ -5,27 +5,34 @@ sidebar_title: Federated graphs
 
 import { VideoFrame } from 'gatsby-theme-apollo-docs'
 
-A **federated graph** is a graph that’s composed of multiple individual **subgraphs**:
-
-[diagram]
+A [federated graph](https://apollographql.com/docs/federation) (also known as a **supergraph** ) is a graph that’s composed of multiple individual **subgraphs**:
+
+```mermaid
+graph BT;
+  gateway(["Supergraph (A + B + C)"]);
+  serviceA[Subgraph A];
+  serviceB[Subgraph B];
+  serviceC[Subgraph C];
+  gateway --- serviceA & serviceB & serviceC;
+```
 
-Each subgraph represents an underlying service or set of services that provide their own GraphQL API. Different subgraphs are often managed by different teams in an organization.
+Each subgraph represents an underlying service or set of services that provide their own GraphQL API. Different subgraphs are often owned by different teams in an organization.
 
-Apollo Studio provides tools to help any number of teams collaborate on a federated graph and its subgraphs.
+Apollo Studio provides tools to help multiple teams collaborate on a federated graph and its subgraphs.
 
 <VideoFrame src="http://placeholder"/>
 
-## Viewing a symbol's originating subgraph
+## Viewing a field's originating subgraph
```

Comment:
Even though we're technically talking "type or field" here, I figure this is clearer than any word shoehorning we could do 😄